itest/tests/exportResults.test.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import fsExtra from "fs-extra"
+
+import dialog from "spectron-fake-dialog"
+import path from "path"
+
+import {
+  click,
+  newAppInstance,
+  pcapIngestSample,
+  startApp,
+  startSearch,
+  writeSearch
+} from "../lib/app"
+import {locator} from "../../src/js/test/integration"
+import {retryUntil} from "../lib/control"
+
+describe("test exporting results", () => {
+  let app
+  let testIdx = 0
+
+  beforeAll(async () => {
+    app = newAppInstance(path.basename(__filename), ++testIdx)
```

Comment:
```suggestion
    await fsExtra.remove("tmp")
    app = newAppInstance(path.basename(__filename), ++testIdx)
```
It's good habit to do "cleaning up" as part of "set up".